src/librustc_typeck/astconv.rs

```diff
@@ -2144,6 +2144,16 @@ impl<'o, 'gcx: 'tcx, 'tcx> dyn AstConv<'gcx, 'tcx> + 'o {
         result_ty
     }
 
+    pub fn is_const_param(&self, expr: &hir::Expr) -> Option<DefId> {
```

Comment:
Conventionally, function names starting with `is_` always return `bool`, so maybe rename this to something else? It also wouldn't hurt to document what the return type means. Maybe something like `const_param_def`?

(same for the function below)

Reply:
Fixed, thanks!